test/integration/etcd/data.go

```diff
@@ -503,13 +503,17 @@ func GetEtcdStorageDataForNamespace(namespace string) map[schema.GroupVersionRes
 		}
 	}
 
+	// add csidrivers
 	// k8s.io/kubernetes/pkg/apis/storage/v1beta1
-	// add csidrivers if CSIDriverRegistry feature gate is enabled
-	if utilfeature.DefaultFeatureGate.Enabled(features.CSIDriverRegistry) {
-		etcdStorageData[gvr("storage.k8s.io", "v1beta1", "csidrivers")] = StorageData{
-			Stub:             `{"metadata": {"name": "csid1"}, "spec": {"attachRequired": true, "podInfoOnMount": true}}`,
-			ExpectedEtcdPath: "/registry/csidrivers/csid1",
-		}
+	etcdStorageData[gvr("storage.k8s.io", "v1beta1", "csidrivers")] = StorageData{
+		Stub:             `{"metadata": {"name": "csid1"}, "spec": {"attachRequired": true, "podInfoOnMount": true}}`,
+		ExpectedEtcdPath: "/registry/csidrivers/csid1",
+	}
+
+	// k8s.io/kubernetes/pkg/apis/storage/v1
+	etcdStorageData[gvr("storage.k8s.io", "v1", "csidrivers")] = StorageData{
+		Stub:             `{"metadata": {"name": "csid2"}, "spec": {"attachRequired": true, "podInfoOnMount": true}}`,
+		ExpectedEtcdPath: "/registry/csidrivers/csid2",
```

Comment:
We need to keep the default GVK to be v1beta for 1 release to handle HA master version skew.

See [here](https://github.com/kubernetes/kubernetes/pull/83474/commits/603a2aa8a99869dc51ea1be2ce6e6ca10c23e55f#diff-a6932c2bd6398cd55df3be4564939a5eR559) and [here](https://github.com/kubernetes/kubernetes/pull/83474/commits/603a2aa8a99869dc51ea1be2ce6e6ca10c23e55f#diff-c74f8b85666a09e8b8a2a052f4a91abdR57)

Reply:
Oops, I see it's added in the 3rd commit. Would still be good to add a TODO and open an issue to track removing it next release.

Reply:
I've updated this to include a TODO, and created issue https://github.com/kubernetes/kubernetes/issues/88821 to track it.